pkgs/by-name/ma/matrix-authentication-service/package.nix

```
@@ -0,0 +1,87 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, buildNpmPackage
+, rustPlatform
+, gnumake
+, open-policy-agent
+}:
+let
+  version = "0.7.0";
+  src = fetchFromGitHub {
+    owner = "matrix-org";
+    repo = "matrix-authentication-service";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-foipChunzRKIbeO+O+XYx0luzaA0G9LKrH59luQl9R0=";
+  };
+  frontend = buildNpmPackage {
+    inherit src version;
+    pname = "matrix-authentication-service-frontend";
+
+    sourceRoot = "source/frontend";
+
+    npmDepsHash = "sha256-ymI+ZkPEGMTLMdTLfKv/v/cgW5iS/nd9PNXFvYaYNjo=";
+
+    installPhase = ''
+      runHook preInstall
+      mkdir -p $out/share
+      mv dist $out/share/matrix-authentication-service-frontend
+      runHook postInstall
+    '';
+  };
+  policies = stdenv.mkDerivation {
+    inherit src version;
+    pname = "matrix-authentication-service-policies";
+    sourceRoot = "source/policies";
+    nativeBuiltInputs = [ gnumake open-policy-agent ];
```

```suggestion
    nativeBuiltInputs = [ open-policy-agent ];
```

Isn't gnumake already part of stdenv?